<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileIsSameDialog</class>
 <widget class="QWidget" name="fileIsSameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>This files are the same file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The source and destination is same</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_size">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_content_size">
         <property name="text">
          <string notr="true">0 KiB</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_modified">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Modified</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_content_modified">
         <property name="text">
          <string notr="true">Today</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_file_name">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>File name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_content_file_name">
         <property name="text">
          <string notr="true">source.txt</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEditNewName">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SuggestNewName">
       <property name="text">
        <string>Suggest new &amp;name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBoxAlways">
       <property name="text">
        <string>&amp;Always do this action</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Rename">
       <property name="text">
        <string>&amp;Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Skip">
       <property name="text">
        <string>&amp;Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
